app/models/invenio_rdm_record_converter.rb

```diff
@@ -441,6 +442,21 @@ def normalize_date(date_string)
   end
 
   def funding(file_id)
-    @@funding_data[file_id] || [{}]
+    funding_sources = @@funding_data[file_id]
+
+    if funding_sources.blank?
+      return []
+    end
+
+    funding_sources.map do |source|
+      source == BLANK_FUNDER_SOURCE
```

Comment:
```suggestion
      source = BLANK_FUNDER_SOURCE
```

Reply:
Oh jeez, there was supposed to be logic here. I think my brain puttered out from being tired around the time I was pushing this up.

Reply:
Okay, this has been updated to the proper `return [] if source == BLANK_FUNDER_SOURCE`

Reply:
Wait, this is incorrect. Return will actually exit the function. Ugh, need to update again.